client/my-sites/site-settings/form-general.jsx

```diff
@@ -400,15 +403,17 @@ module.exports = React.createClass( {
 				<Card>
 					<form onChange={ this.markChanged }>
 						{ this.siteOptions() }
+						{ this.blogAddress() }
 						{ this.languageOptions() }
 						{ this.holidaySnowOption() }
 					</form>
 				</Card>
-				<SectionHeader label={ this.translate( 'Address and visibility' ) }>
+				<SectionHeader label={ this.translate( 'Visibility' ) }>
 					<Button
```

Comment:
This is an added bonus - because Jetpack sites do not require the custom address controls, so the 'Address and Visibility' label is a bit confusing there. Now it will be just 'Visibility' . Win win!